examples/favorites_advanced/bloc_favorites_advanced/lib/base/ui_components/puppies_app_bar.dart

```diff
@@ -31,7 +30,8 @@ class PuppiesAppBar extends StatelessWidget implements PreferredSizeWidget {
   ) =>
       AppBarTextField(
         searchContainerColor: Colors.blue,
-        title: Text(item.type.asTitle()),
+        // title: Text(item.type.asTitle()),
+        title: _createTitle(context),
```

Comment:
@StanislavPrime , try to stick to the convention of building stuff. In this case, the better name for  `_createTitle` would be `_buildTitle`

Reply:
I changed the name of the method  `_createTitle` to `_buildTitle`